src/Engine/Graphics/Renderer/IOverlayRenderer.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <Library/Geometry/Size.h>
+
+class IOverlayRenderer {
+ public:
+    virtual ~IOverlayRenderer() = default;
+    virtual void render(const Sizei& outputPresent, int &drawCalls) = 0;
```

Comment:
We generally use pointers for output parameters. This way it's clear at the call site that we're passing in an output parameter.